api/templates/design.go

```diff
@@ -1027,8 +1031,9 @@ type ExampleVersionData struct {
 
 type ExampleVersionDetailData struct {
 	ExampleVersionData
-	Components []string `json:"components" yaml:"components"`
-	Readme     string   `json:"readme" yaml:"readme"`
+	Components      []string `json:"components" yaml:"components"`
+	LongDescription string   `json:"longDescription" yaml:"longDescription"`
+	Readme          string   `json:"readme" yaml:"readme"`
```

Comment:
`LongDescription` se ukládá do folderu templaty jako `description.md`. Funguje to obdobně jako `Readme` (`README.md`), tak jsem hlavně kopíroval. Je tam jediný rozdíl, že to není povinný. Respektive nechceme po releasu všechny existující templaty honem upravovat aby nezačaly failovat.

Reply:
Tohle je definice API pro https://goa.design/ framework a v podstatě jsem jen hledal použití `readme` a kopíroval to podle něj. (A ve zbytku kódu vlastně taky. 😃)